app/models/user.py

```diff
@@ -0,0 +1,24 @@
+from app import db
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+
+class User(db.Model):
+    id = db.Column(db.BigInteger, primary_key=True, autoincrement=True)
```

Comment:
nice one on setting the ID to be BigInteger, meaning that you can expect that we can have a LOT of users. But would the DB size and performance be compromised? Will it need to be scaled up?

Reply:
For now it shouldn't be. Maybe in the future, when there are many users, there will be some adjustments